.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -27,6 +27,19 @@ else
     echo "Running tests: Go files changed"
 fi
 
+
+$BRANCHNAME=$(echo -n "/auto-pr-${pr_number}"| base64)
+curl --header "Accept: application/json" --header "Authorization: Bearer $TEAMCITY_TOKEN" https://ci-oss.hashicorp.engineering/app/rest/builds/multiple/branch:$BRANCHNAME,defaultFilter=false --request POST --header "Content-Type:application/xml" --data-binary @/teamcitycancelparams.xml -o result.json
```

Comment:
It looks like this is using https://www.jetbrains.com/help/teamcity/rest/buildapi.html#cancelMultiple.

If I understand correctly, this will cancel any running builds on the current branch. I think that might include non-VCR builds (i.e. RECORDING builds) which may not be safe to cancel, since it could leave dangling resources?